spec/validation/loader_spec.rb

```diff
@@ -49,19 +49,17 @@
 
     let :filesystem do
```

Comment:
I was initially a bit unsure about using this method as I thought it might be a bit too magic, but actually this is quite nice and avoids having to remember to wrap each test case in a `filesystem.test` block. Thanks for the suggestion, made in https://github.com/alces-software/metalware/commit/691ac8706964c7b5fd0220e30d8e82bcc097d012.

Reply:
The use of the `root_setup` is a bit clunky. Every now and then I forget to put it in a `:before` block which can cause some weird errors depending on which order the tests run in. This could probably be improved in the future.

Reply:
Yep, there's a few things we could still improve about the `FileSystem`/`FakeFS` etc. stuff - it's much better than it used to be however and using `root_setup` here is an improvement IMO :slightly_smiling_face:.